### ISSUE TITLE
Install Error (pip and conda)

### ISSUE THREAD

**Issue by Shigeyuki-Matsunami** [JeschkeLab/DeerLab#263]
**Describe the bug**

Hi,
I tried both pip install and conda install of Deerlab, 
but both of them gave me the following error and I could not install.

My enviroment and version are

OS: Windows 10
IDE: Anaconda (python 3.9)

Please let me know if there is anything I should check.

best,
matsunami

**1) Error of pip install** 
-------------------------------------------------------------------------
```
(ESR) C:\Users\MATSU>python -m pip install deerlab
Collecting deerlab
  Using cached DeerLab-0.13.2.tar.gz (112 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "string", line 2, in  "module"
        File "pip-setuptools-caller", line 34, in  "module"
        File "C:\Users\MATSU\AppData\Local\Temp\pip-install-2gi50t2p\deerlab_d7699b9cbf7f41cda41871df73e53b77\setup.py", line 86, in "module"
          long_description=open('README.md').read(),
      UnicodeDecodeError: 'cp932' codec can't decode byte 0x93 in position 2643: illegal multibyte sequence
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.
```

**2) Error of conda install**
---------------------------------------------------------------------------
```
(ESR) C:\Users\MATSU>conda install deerlab -c JeschkeLab
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
                                                                                                                       -failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions
```

**Comment by luisfabib** [comment]
Hi Matsunami! Thanks for the report. The `pip` error seems to be a simple encoding error. There must be some Unicode character in the `README` file that is not supported by your computer's encoding. This is a typical error. 

I will push a PR to fix this in a few moments. For now, you can fix it locally on your system by:
1) Go to `C:\Users\MATSU\AppData\Local\Temp\pip-install-2gi50t2p\deerlab_d7699b9cbf7f41cda41871df73e53b77`
2) Open the `setup.py` file with a text editor.
3) Change line 86 from: 
    ```
    long_description=open('README.md').read(),
    ```
    to 
    ```
    long_description=open('README.md', encoding='utf-8').read(),
    ```
4) On the terminal/console try again the installation
    ```
    cd C:\Users\MATSU\AppData\Local\Temp\pip-install-2gi50t2p\deerlab_d7699b9cbf7f41cda41871df73e53b77 
    python -m setup.py install
    ```

**Comment by luisfabib** [comment]
Regarding the `conda` error, it seems that it is not being able to find a compatible environment to install DeerLab. There are some possible approaches out there that might fix this.  

- Update all packages 
```
conda update --all --yes
```
- Disable channel priority
```
conda config --set channel_priority false
```
- Create and activate a separate environment
```
conda create --name myenv
conda activate myenv
```

As far as I can think from the error you are getting this is not caused by DeerLab itself.

**Comment by luisfabib** [comment]
Let me know if these approaches work for you. Otherwise, the issue will be closed as soon as PR #264  is merged, solving the `pip` installation error.

**Comment by Shigeyuki-Matsunami** [comment]
Hi, Luis! 
Thank you for your kindness., and the pip installation worked fine, it's great.
However, conda installl, I tried to create a new environment according to your procedure, but it still  doesn't work.

Anyway, now that I've installed the DeerLab, I'd like to explore it.
